protocol/messenger_contacts.go

```diff
@@ -1291,3 +1304,23 @@ func (m *Messenger) forgetContactInfoRequest(publicKey string) {
 
 	delete(m.requestedContacts, publicKey)
 }
+
+func (m *Messenger) GetDeleteForMeMessages() ([]*protobuf.SyncDeleteForMeMessage, error) {
+	return m.persistence.GetDeleteForMeMessages()
+}
+
+func (m *Messenger) SubscribeToSelfContactChanges() chan *SelfContactChangeEvent {
+	s := make(chan *SelfContactChangeEvent, 10)
```

Comment:
Why 10?

Reply:
Well I saw we mostly set it to `100`. But I thought that this one has relatively low usage and there's no need for such big buffer. Do you think it's better to keep it larger?

Reply:
No, I just forgot how `chan` works and I found it weird to see a random magic number. I don't know what's best here

Reply:
10 should be enough imo. And we actually only use this subscription in tests anyway.